crates/typos-cli/src/bin/typos-cli/args.rs

```diff
@@ -57,6 +57,10 @@ pub(crate) struct Args {
     #[command(flatten, next_help_heading = "Config")]
     pub(crate) config: ConfigArgs,
 
+    /// Respect excluded files even for paths passed explicitly.
+    #[arg(long, group = "mode", help_heading = "Mode")]
```

Comment:
Why is this in the `mode` group and under the `Mode` heading?

Reply:
Please indicate the `group` and `help_heading` you prefer.

Reply:
It doesn't need to be in any group.  In fact, I expect the use of the group breaks the pre-commit hook because this declared `--write-changes` and `--force-exclude` as mutually exclusive.

As for a help_heading, let's leave this off for now, leaving it to `Options`

Reply:
Thanks for the explanations. I removed the `group` and `help_heading` without moving the `arg` position, which means `--force-exclude` appears at the end of `Config` group. Let me know if you prefer otherwise (it would appear below the `-j` option if I move it up).

Reply:
Ah, I hadn't realized some config comes before this.  Please go with `help_heading = None` then

Reply:
I updated the PR. Is the output what you expected?

Reply:
Seems good enough